NitroxClient/GameLogic/EquipmentSlots.cs

```diff
@@ -34,19 +42,26 @@ public void BroadcastEquip(Pickupable pickupable, GameObject owner, string slot)
         else
         {
             // UWE also sends module events here as they are technically equipment of the vehicles.
-            ModuleAdded moduleAdded = new(itemId, ownerId, slot);
-            packetSender.Send(moduleAdded);
+            string classId = pickupable.RequireComponent<PrefabIdentifier>().ClassId;
+            NitroxTechType techType = pickupable.GetTechType().ToDto();
+            EntityMetadata metadata = entityMetadataManager.Extract(pickupable.gameObject).OrNull();
+
+            InstalledModuleEntity moduleEntity = new(slot, classId, itemId, techType, metadata, ownerId, []);
+
+            if (packetSender.Send(new EntitySpawnedByClient(moduleEntity, true)))
+            {
+                Log.Debug($"Sent: Added module {pickupable.GetTechType()} ({itemId}) to equipment {owner.GetFullHierarchyPath()} in slot {slot}");
```

Comment:
How spam-y is this logging?

Reply:
Not very, it only triggers if you add or remove modules from a vehicle which is fairly uncommon. It's also a debug message so most people won't see it.